lib/src/model/robotoff_nutrient_extraction.dart

```diff
@@ -0,0 +1,201 @@
+import 'package:json_annotation/json_annotation.dart';
+import '../interface/json_object.dart';
+
+part 'robotoff_nutrient_extraction.g.dart';
+
+@JsonSerializable()
+class RobotoffNutrientExtractionResult extends JsonObject {
```

Comment:
Generally speaking, I would prefer one file per class (I'm from Java) and an explicit `Robotoff` prefix for all class names.
Not the most important part of my review, though.